vermouth/data/force_fields/martini30dev/aminoacids.ff

```diff
@@ -0,0 +1,859 @@
+   BB   SC1  SC3       2   120.000   60.0 
+
+[dihedrals]
+;  i     j    k    l   funct   angle  force.c.
+ SC4   SC2  SC3  SC1       2    180.0    50.0     ; to prevent backflipping of ring
+
+[exclusions]
+; We do not want non-bounded interactions within the residue.
+BB SC1 SC2 SC3 SC4
+SC1 SC2 SC3 SC4
+SC2 SC3 SC4
+SC3 SC4
+
+;;; TRYPTOPHAN
+
+[ moleculetype ]
+;molname       nrexcl
+TRP                1
+
+[ atoms ]
+;id type resnr residu atom cgnr   charge
+1   $prot_default_bb_type       1     TRP     BB     1    0
+2   TC3      1     TRP     SC1    2    0       36
+3   TN3d     1     TRP     SC2    3    0       36
+4   TC4      1     TRP     SC3    4    0        0
+5   TC4      1     TRP     SC4    5    0       36
+6   TC4      1     TRP     SC5    6    0       36
+
+[ bonds ]
+#meta {"group": "Side chain bonds"}
+;  i     j   funct   length  force.c.
+  BB   SC1    1       0.315     5000 	
+#meta {"group": "Side chain bonds", "ifdef": "FLEXIBLE"}
+;  i     j   funct   length  
+ SC1   SC2    1       0.335  $stiff_fc
+ SC2   SC5    1       0.412  $stiff_fc
+ SC4   SC5    1       0.293  $stiff_fc
+ SC1   SC4    1       0.404  $stiff_fc
+ SC2   SC4    1       0.470  $stiff_fc
+
+[ constraints ]
+#meta {"group": "Side chain bonds", "ifndef": "FLEXIBLE"}
+;  i     j   funct   length  
+ SC1   SC2    1       0.335
+ SC2   SC5    1       0.412
+ SC4   SC5    1       0.293
+ SC1   SC4    1       0.404
+ SC2   SC4    1       0.470
+
+[ angles ]
+#meta {"group": "Side chain angles"}
+;  i     j    k     funct   angle  force.c.
+  BB   SC1  SC2       2   120.000   60.0 
+  BB   SC1  SC4       2   130.000    60.0  
+
+[ dihedrals ]
+;  i     j    k    l   funct   angle  force.c.
+ SC5   SC4  SC2  SC1       2   180.0    100.0
+
+[ virtual_sitesn ]
+SC3 SC5 SC4 SC2 SC1 -- 2
+
+[exclusions]
+; We do not want non-bounded interactions within the residue.
+BB SC1 SC2 SC3 SC4 SC5
+SC1 SC2 SC3 SC4 SC5
+SC2 SC3 SC4 SC5
+SC3 SC4 SC5
+SC4 SC5
+
+;;; Links
+
+;; Links for COIL. We apply them first as coil is the default.
+[ link ]
+resname $protein_resnames
+[ bonds ]
+BB +BB 1 0.350 1250 {"group": "Backbone bonds"}
+
+[ link ]
+resname $protein_resnames
+[ features ]
+scfix
+[ molmeta ]
+scfix true
+[ dihedrals ]
+#meta {"group": "SC-BB-BB-SC scFix"}
+SC1 BB +BB +SC1 1 dihphase(SC1,BB,+BB,+SC1|.0f) 75 1
+
+[ link ]
+resname $protein_resnames
+[ features ]
+scfix
+[ molmeta ]
+scfix true
+[ angles ]
+SC1 BB +BB 10 100 15 {"group": "SC-BB-BB and BB-BB-SC scFix", "comment": "SC-BB-BB"}
+
+[ link ]
+resname $protein_resnames
+[ features ]
+scfix
+[ molmeta ]
+scfix true
+[ angles ]
+BB +BB +SC1 10 100 15 {"group": "SC-BB-BB and BB-BB-SC scFix", "comment": "BB-BB-SC"}
+
+[ link ]
+resname $protein_resnames
+[ angles ]
+-BB  BB  SC1  2 100 25 {"group": "BBS angles regular martini", "version": 1}
+
+[ link ]
+resname $protein_resnames
+[ angles ]
+#meta {"group": "First SBB regular martini"}
+SC1 BB +BB 2 100 25 {"version": 1}
```

Comment:
Doesn't this clobber the scfix one?

Reply:
No because of the "version" attribute. All interactions are version 0 by default. The clobbering only works for a given version.